### ISSUE TITLE
RadListView itemHold event is not triggered on iOS

### ISSUE THREAD

**Issue by felix-idf** [telerik/nativescript-ui-feedback#473]
### Did you verify this is a real problem by searching [Stack Overflow](http://stackoverflow.com/questions/tagged/nativescript)?
_Yes_

### Tell us about the problem
The itemHold event is not triggered on iOS. It works as expected on Android.

### Which platform(s) does your issue occur on?
_iOS (tested on iOS 8.4 and 10.3)_

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** 3.3.0
- **CLI:** 3.3.1
- **Cross-platform modules:** 3.3.0
- **Runtime(s):** tns-ios: 3.3.0

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
_No_

**Comment by tsonevn** [comment]
Hi @felix-idf,
Please provide some more info about your case and how we could recreate this behavior. It would help if you could send us sample project, which could be debugged locally.

**Comment by felix-idf** [comment]
You just need to add a RadListView to the project in xml...

```
<lv:RadListView items="{{ items }}" itemHold="onItemHold">
    <lv:RadListView.listViewLayout>
        <lv:ListViewLinearLayout scrollDirection="Vertical" />
    </lv:RadListView.listViewLayout>
    <lv:RadListView.itemTemplate>
        <StackLayout>
            <!-- your template -->
        </StackLayout>
    </lv:RadListView.itemTemplate>
</lv:RadListView>
```

... and try to log something from the itemHold event
```
exports.onItemHold = function(args) {
    console.log("this event is raised only on android, not on ios");
}
```

This works for Android, but not on iOS

**Comment by tsonevn** [comment]
Hi @felix-idf,
Thank you for the provided code snippet.
I tested it on my side and have to confirm that the this is a real issue for iOS and the `itemHold` event will not be fired. I am logging this issue as a bug and would suggest keeping track on it for further info.

**Comment by dexter2019** [comment]
any update on this ??

**Comment by dexter2019** [comment]
@manojdcoder  even the itemselectedevent is not working. did it ever work for you ?

**Comment by manojdcoder** [comment]
@dexter2019 Do you mean adding longPress prevents item selected event? In my case, I maintain my own flag and stylings for selection.

**Comment by dexter2019** [comment]
@manojdcoder no. I meant I have a radlist view, and itemselected event is not getting fired.

**Comment by johandrott** [comment]
Any progress on this? The workaround described by @dexter2019 doesn't work for me. Setting the longPress event on the item template prevents the itemTap event from being triggered on android. If I set both tap and longPress event on the item template all works fine on android, but iOS triggers the tap event if doing a longPress.

**Comment by jelledijkstra97** [comment]
Any progress on this issue? I'm having the exact same problem as johandrott described. The RadListView is completely unusable now. Please fix it.
